mitosheet/mitosheet/public/v1/sheet_functions/types/to_int_series.py

```diff
@@ -7,7 +7,7 @@
 For going to a number series.
 """
 from typing import Any
-from mitosheet.sheet_functions.types.to_float_series import to_float_series
+from mitosheet.public.v1.sheet_functions.types.to_float_series import to_float_series
```

Comment:
It's kinda interesting that we're versioning things like `to_int_series`. We've previously added new functionality to these and didn't version them. We thought about it as an improvement. It makes sense that we're versioning everything now since we're doing a big sheet functions overall. But if we decide in 3 weeks to make our `to_int_series` handle more formats, we probably don't need to version the interface then. What do you think?

Reply:
I added some clarification to the README. Generally, if we're _extending_ behavior, we don't need to upgrade. It's only backwards incompatible changes. 

Our type casting / sheet functions are so different now that it makes sense to cut it here, methinks!